puzzlegenerator/puzzlegenerator_ui/src/main/java/com/flurnamenpuzzle/generator/ui/view/TargetPathSelectionCard.java

```diff
@@ -43,8 +43,8 @@ public void actionPerformed(ActionEvent e) {
 				int returnVal = fileChooser.showOpenDialog(null);
 				// check if user has selected a file
 				if (returnVal == JFileChooser.APPROVE_OPTION) {
-					File selectedFodler = fileChooser.getCurrentDirectory();
-					String pathOfSelectedFolder = selectedFodler.getPath();
+					File selectedFolder = fileChooser.getSelectedFile();
```

Comment:
In initializeComponents() befindet sichr folgender Code:
this.fileChooser.setFileSelectionMode(JFileChooser.DIRECTORIES_ONLY);